troposphere/static/js/components/common/InstanceAccessListView.jsx

```diff
@@ -0,0 +1,199 @@
+            instance_ip = this.props.instance.get('ip_address'),
+            current = UserStore.getUsername(this.state.username);
+        if(options == null) {
+            return (<div className="loading" />);
+        }
+        return (
+        <div>
+            <h3>Create an Instance Access Request</h3>
+            <div>
+              <p>
+                {"Here we will describe the instance access request process like a human. "} <br/><br/>
+                {"Requirements include:"}<br/>
+                {"- Instance access will allow the shared user to login via password (if available) and SSH key access, to the instance: "+instance_ip}<br/>
+                {"- Multiple instance access request(s) can be created, but users must be invited one at a time."}<br/>
+                {"- The shared user will be notified in the application when you send an instance access request."}<br/>
+                {"- At any point after the access request is approved, you can delete the request and the users login/SSH access will also be removed from the application."}<br/>
+                </p>
+            </div>
+            <div className="form-group">
+                <label htmlFor="instanceRequest" className="col-sm-3 control-label">
+                    {"Username to Share Instance With"}
+                </label>
+                <div className="col-sm-9">
+                <ChosenSingleDropdown current={current}
+                    className="form-control"
+                    searchField={"username"}
+                    models={options}
+                    onModelSelected={this.onSelectUser}
+                />
+                </div>
+            </div>
+        </div>);
+    },
+    updateRequestsList: function() {
+        let {InstanceAccessStore} = this.props.subscriptions;
+
+        InstanceAccessStore.clearCache();
+    },
+    getRequests: function() {
+        let {InstanceAccessStore} = this.props.subscriptions,
+            current_username = context.profile.get('username'),
+            access_requests;
+
+        if(this.props.instance) {
+            access_requests = InstanceAccessStore.getForInstance(this.props.instance);
+        } else {
+            access_requests = InstanceAccessStore.getAll();
+        }
+        if(access_requests) {
+            access_requests = access_requests.cfilter(access_request => {
+                if(access_request.get('status') == 'denied' && access_request.get('instance').user != current_username) {
+                    return false;
+                }
+                return true;
+            });
+        }
+        return access_requests;
+    },
+    renderHeaderRow: function() {
+        return (
+            <tr>
+                <th style={{ width: "300px" }}>Instance</th>
+                <th style={{width: "150px" }}>User</th>
+                <th style={{ width: "75px" }}>Status</th>
+                <th style={{textAlign: "right" }}>Actions</th>
+            </tr>
```

Comment:
Might want to change the dimensions here. This is what it looks like fullscreen (underneath `Instance Access Requests` ):
<img width="1157" alt="screen shot 2017-11-21 at 3 39 48 pm" src="https://user-images.githubusercontent.com/2889930/33100610-41917988-ced2-11e7-9979-5ac6269de293.png">